source/extensions/common/wasm/wasm.cc

```diff
@@ -93,7 +93,7 @@ Wasm::Wasm(WasmHandleSharedPtr base_wasm_handle, Event::Dispatcher& dispatcher)
                [&base_wasm_handle]() {
                  return createWasmVm(absl::StrCat(
                      "envoy.wasm.runtime.",
-                     toAbslStringView(base_wasm_handle->wasm()->wasm_vm()->runtime())));
+                     toAbslStringView(base_wasm_handle->wasm()->wasm_vm()->getEngineName())));
```

Comment:
Since this is wasm's API change and not ours, I see no issue with proceeding. I'm not clear if wasm is still labeled as beta anyways.

Reply:
This isn't an user-facing change.